@here/harp-mapview/test/MapViewTest.ts

```diff
@@ -266,6 +290,20 @@ describe("MapView", function () {
                     heading: 30
                 }
             },
+            {
+                testName: "large polygon bounds + angles, off-center projection",
+                lookAtParams: {
+                    bounds: new GeoPolygon([
+                        new GeoCoordinates(40.0, 13.0),
+                        new GeoCoordinates(40.0, 20.0),
+                        new GeoCoordinates(52.0, 20.0),
+                        new GeoCoordinates(52.0, 13.0)
+                    ]),
+                    tilt: 80,
+                    heading: 30
+                },
+                ppalPoint: { x: -0.8, y: 0.1 }
```

Comment:
please add a test at the extremes, i.e. -1, 1, and make sure they don't break etc.

Reply:
done